specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetworkGateway.json

```diff
@@ -1167,6 +1176,12 @@
           }
         ],
         "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkGatewayConnection"
+            }
```

Comment:
same

Reply:
For all hybrid networking resources, we return the KVS object being updated from NRP. I have kept the behavior same here as well.

The NRP behavior is designed in the same way as well.